src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.Value.cs

```diff
@@ -145,71 +145,6 @@ public IAbstractAnalysisValue GetNegatedValue()
                 return false;
             }
 
-            private static bool TryDecodeOSVersion(
```

Comment:
Just moved out of the `private readonly struct PlatformMethodValue` to use in `PlatformCompatibilityAnalyzer.OperationVisitor.cs`